README.md

```diff
@@ -8,9 +8,11 @@ This is an initiative to visualize [Project Bedrock](https://github.com/microsof
 
 Currently, Spektate consists of a command line interface and a simple dashboard prototype. The instructions to use both are below.
 
+Note: Spektate dashboard will delete deployments when their corresponding builds/releases have expired in Azure DevOps.
+
```

Comment:
nit: should we mention something like "as of january 2020 this is every 2 weeks?" or a link to the expiration policy, if any exists?

Reply:
@edaena I can't seem to find the page where they specify it, but i'll continue looking for it